build/index.ts

```diff
@@ -159,7 +160,7 @@ function injectSource(doc, document, metadata) {
 export interface BuiltDocument {
   doc: Doc;
   liveSamples: any;
-  fileAttachments: any;
+  fileAttachments: Map<string, string>;
```

Comment:
To avoid surprises, let's also rename this variable:

```suggestion
  fileAttachmentMap: Map<string, string>;
```

We can then return a property by the same name here:

https://github.com/mdn/yari/blob/73ded5a3b6ff4e601fab93b0768bb2b1f3051ccd/build/index.ts#L552